composer.json

```diff
@@ -34,7 +34,8 @@
     "phpunit/phpunit": "^5.7 || ^6.5 || ^7.5",
```

Comment:
@spacedmonkey My understanding was that we can now remove this line here, as the wp-test-utils library will automatically install it.